TelCo.ColorCoder/ColorCodeIdentifier.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+namespace TelCo.ColorCoder
+{
+    public class ColorCodeIdentifier
```

Comment:
Class Name ColorCodeIdentifier holds multiple responsibilities....is this class  identify ColorCode only  ?

Reply:
Updated the class names as per the review comments and committed to the master branch